src/Monitor/Monitor/ChangeLog.md

```diff
@@ -29,6 +29,9 @@
 * Use common alert schema enabled for the receivers. This is not applicable for SMS, Azure App push , ITSM and Voice recievers
 * Webhooks now supports Azure active directory authentication .
 
+## Version 1.3.1
```

Comment:
Just add this into the 'Upcoming Release' section.

Reply:
+1